cards/financial-professional-location/template.hbs

```diff
@@ -1,10 +1,13 @@
+{{! Some partials here use the result.ordinal parameter, a reserved value automatically provided to
+    any card template. This value contains the ordinal of the card.
+}}
+
 <div class="HitchhikerFinancialProfessionalLocation HitchhikerLocationCard {{cardName}}">
   {{> image }}
   <div class="HitchhikerFinancialProfessionalLocation-body">
     {{#if (any card.title card.distance)}}
     <div class="HitchhikerFinancialProfessionalLocation-topRow">
-      {{> ordinal }}
-      {{> title }}
+      {{> ordinalAndTitle displayOrdinal=(all card.showOrdinal result.ordinal) }}
       <div class="HitchhikerLocationCard-closeCardButtonWrapper">
         {{> closeCardButton }}
         {{> distance }}
```

Comment:
Should distance be part of `closeCardButtonWrapper` div? wouldn't this look a bit weird if both distance and closeCardButton is display on the card?

Reply:
yes, I think it would look weird. but I think we would want both of these to be right-aligned. so maybe distance should go before the button and the wrapper should have flex styling?

Reply:
looks like the button is not configurable? seems to have css display: none by default. Perhaps you could checkin with rose if this needs to be fix. And regard to the distance alignment, maybe she would have an opinion between the template changes to be same as location-standard card, or css styling changes only, which is what I'm kind of leaning toward instead of merging ordinal and title

Reply:
Since `distance` is already part of `closeCardButtonWrapper`, I think that's out of scope of this QA item. As for the DOM changes, no CSS classes are being removed in this PR which means that most styling that is currently targeting them wouldn't break during an upgrade. So I'd say handling this with CSS only changes isn't critical